content/fast/minify-css/index.md

```diff
@@ -55,104 +55,115 @@ In practice, you would like to introduce this technique as one of the steps of y
+1. Open the project [in Glitch](https://glitch.com/~fav-kitties-animated).
+1. Click “View Source”, and then, click on the name of the project in the upper left side.
+1. In the drop-down menu, choose “Remix and Edit” 🎤, to make changes on the project.
+1. Once in the cloned project, go to the “Tools” option in the lower left corner of the edit view, and click on “Console”.
 
 To minify the resulting CSS, you’ll use the [optimize-css-assets-webpack-plugin](https://github.com/NMFR/optimize-css-assets-webpack-plugin): 
 
-1. In the console, run `npm install --save-dev optimize-css-assets-webpack-plugin`.
+1. In Glitch console, run `npm install --save-dev optimize-css-assets-webpack-plugin`.
 1. Run `refresh`, so the changes are synchronized with the Glitch editor.
 
-Next, go back to the Glitch editor, and open the **webpack.config.js** file, and make some modifications:
+Next, go back to the Glitch editor, open the **webpack.config.js** file, and make the following modifications:
 
-At the beginning of the file, add:
+Load the module at the beginning of the file:
 ```
 const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 ```
 
-Inside **module.exports**, insert:
+Then, pass an instance of the plugin to the **plugins** array:
 ```
-optimization: {
-	minimizer: [
-		new OptimizeCSSAssetsPlugin({})
-	]
-}
+  plugins: [
+    new HtmlWebpackPlugin({template: "./src/index.html"}),
+    new MiniCssExtractPlugin({filename: "[name].css"}),
+    new FixStyleOnlyEntriesPlugin(),
+    new OptimizeCSSAssetsPlugin({})
+  ]
 ```
 After making the changes a rebuild of the project will be triggered.
 This is how the resulting **webpack.config.js** will look like:
 
 <div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
   <iframe
     allow="geolocation; microphone; camera; midi; vr; encrypted-media"
-    src="https://glitch.com/embed/#!/embed/minify-css-optimized?path=webpack.config.js&previewSize=0"
-    alt="minify-css-optimized on Glitch"
+    src="https://glitch.com/embed/#!/embed/fav-kitties-animated-min?path=webpack.config.js&previewSize=0"
+    alt="fav-kitties-animated-min on Glitch"
     style="height: 100%; width: 100%; border: 0;">
   </iframe>
 </div>
 
-1. Go back to the console, and run `cd public`.
-1. Run `vi main.css`, to inspect the content of the CSS.
+Next, you'll check the result of this optimization with performance tools.
 
-You'll find a more compact version of the CSS, after minification has been applied.
+## Monitor
 
-<div class="aside note">
-Note: This demo is based on Webpack 4. It's likely that Webpack 5 will come with a CSS minimizer built-in. If that’s the case you won’t need to configure a plugin as done in the previous step.
-</div>
+First, run Lighthouse on the [optimized site](https://fav-kitties-animated-min.glitch.me/). 
 
-## Monitor
+The report doesn't show "Minify CSS" as "Opportunity" anymore, and has now moved to "Passed Audits" section:
```

Comment:
Subject of the sentence shifts from first half to the second half.  Maybe:

"Minify CSS" doesn't appear under "Opportunities" anymore; it's moved to the "Passed Audits" section: